styx-scheduler-service/src/main/java/com/spotify/styx/state/QueuedStateManager.java

```diff
@@ -101,7 +103,6 @@ public void initialize(RunState runState) throws IsClosed {
     final long counter;
     try {
       counter = storage.getLatestStoredCounter(workflowInstance).orElse(NO_EVENTS_PROCESSED);
-      storeActivation(workflowInstance, counter);
```

Comment:
So we store activation later in `process` after storing event, is that correct? If yes, we seem missing one `counter` value. Does that matter?